src/cascadia/TerminalSettingsEditor/AISettings.xaml

```diff
@@ -126,23 +134,106 @@
+                                        HorizontalAlignment="Center">
+                                    <TextBlock x:Uid="AISettings_ClearOpenAIKey" />
+                                </Button>
+                                <Button Click="SetOpenAIActive_Click"
+                                        HorizontalAlignment="Center"
+                                        Visibility="{x:Bind mtu:Converters.InvertedBooleanToVisibility(ViewModel.OpenAIActive), Mode=OneWay}">
+                                    <TextBlock x:Uid="AISettings_SetActiveProvider" />
+                                </Button>
+                            </StackPanel>
+                        </Grid>
+                        <Grid RowSpacing="8"
+                              Visibility="{x:Bind mtu:Converters.InvertedBooleanToVisibility(ViewModel.IsOpenAIKeySet), Mode=OneWay}">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
+                            <StackPanel Grid.Row="0"
+                                        Padding="16"
+                                        CornerRadius="4"
+                                        Orientation="Horizontal"
+                                        Spacing="14">
+                                <StackPanel.Background>
+                                    <SolidColorBrush Opacity="0.7"
+                                                     Color="#404040" />
+                                </StackPanel.Background>
+                                <muxc:InfoBadge x:Name="OpenAIDescriptionInfoBadge"
+                                                VerticalAlignment="Top"
+                                                FontSize="16"
+                                                Style="{ThemeResource AttentionIconInfoBadgeStyle}">
+                                    <muxc:InfoBadge.IconSource>
+                                        <muxc:FontIconSource FontSize="16"
+                                                             Glyph="&#xE625;" />
+                                    </muxc:InfoBadge.IconSource>
+                                </muxc:InfoBadge>
+                                <RichTextBlock TextWrapping="WrapWholeWords"
+                                               MaxWidth="900">
+                                    <Paragraph>
+                                        <Run x:Name="AISettings_OpenAIDescriptionPart1" /><Hyperlink NavigateUri="https://go.microsoft.com/fwlink/?linkid=2251839"

```

Comment:
For the OpenAI case, would we want to / need to link out to their terms?

Reply:
I believe @nguyen-dows said he'll hunt down the ToU once we get closer to release!